man/chpldoc.txt

```diff
@@ -35,7 +35,8 @@ DESCRIPTION
 
   --save-sphinx <directory>  Save generated Sphinx project in directory.
 
-  --text-only       Generate text based documentation instead of HTML. Takes precedence over --[no-]html
+  --text-only       Generate text based documentation instead of HTML. Takes
```

Comment:
IMO, "text based" should be "text-based", IMO (rationale: it acts as one word; it avoids the potential parsing ambiguity of reading the three words as "text based-documentation").